code_annotations/base.py

```diff
@@ -524,69 +529,26 @@ def check_group(self, annotations):
                 if token not in found_tokens:
                     self._add_annotation_error(
                         annotations[0],
-                        "missing non-optional annotation: '{}'".format(token)
-                    )
-
-    def check_annotation(self, annotation, current_group):
```

Comment:
Wow looks like this was just completely unused, eh?

Reply:
Yes, I had forgotten to remove it after my previous cleanup.